src/content/en/tools/chrome-devtools/_toc.yaml

```diff
@@ -53,8 +53,8 @@ toc:
       section:
       - title: Get Started
         path: /web/tools/chrome-devtools/javascript/
-      - title: Set Breakpoints
-        path: /web/tools/chrome-devtools/javascript/add-breakpoints
+      - title: Breakpoints Guide
```

Comment:
Good question. My reasoning is that, with the current task-based titles, you don't know what type of doc you're getting. It's annoying to get a reference when you were hoping for a tutorial. However, this new approach could be a mistake, for all the reasons that task-based names are more helpful.

Also, the directories are still task-based. E.g. "Breakpoints Guide" is nested in "Inspect and Debug JavaScript". But that's not helpful if you're coming from Search.

Style Guide doesn't give recommendations one way or the other on doc titles.

Either way, a better task-based title would be "Pause Your Code With Breakpoints". "Set Breakpoints" and "Breakpoints Guide" are both feature-based: neither's useful if you don't know what a breakpoint is.

Let's leave this for now but I can revisit in a later PR